packages/contracts-ops/contracts/Config.sol

```diff
@@ -193,25 +240,60 @@ abstract contract Config {
 
     function bridgeTokenUpgrade(string memory domain)
         public
+        override
         onlyInitialized
         returns (Upgrade memory)
     {
         return
             abi.decode(loadBridgeAttribute(domain, "bridgeToken"), (Upgrade));
     }
 
+    function tokenRegistry(string memory domain)
+        public
+        override
+        onlyInitialized
+        returns (TokenRegistry)
+    {
+        return TokenRegistry(address(tokenRegistryUpgrade(domain).proxy));
+    }
+
     function tokenRegistryUpgrade(string memory domain)
```

Comment:
nit: should be above func that uses it

Reply:
https://github.com/nomad-xyz/monorepo/pull/484/commits/2d50e92c61b3ddbf5e85a9125d8204ef1ed675eb